ui/imports/shared/stores/BIP39_en.qml

```diff
@@ -12,8 +12,10 @@ ListModel {
             if (xhr.readyState === XMLHttpRequest.DONE) {
```

Comment:
We have `StringUtilsInternal::readTextFile(filePath)` in C++, just saying :)

Reply:
good point! Will also remove all the security warnings 💪